BackofficeBundle/Resources/public/coffee/table/TableviewAction.coffee

```diff
@@ -7,13 +7,13 @@ TableviewAction = OrchestraView.extend(
     @options = options
     _.bindAll this, "render"
     @loadTemplates [
-      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/tableviewActions'
+      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/table/tableviewActions'
```

Comment:
Question: is it really worth to keep the prefix table now that the file is in a table folder?